--- forgejo/webpack.config.js.webpack-mock-crash	2025-05-27 14:45:25.013906567 +0200
+++ forgejo/webpack.config.js	2025-05-27 15:00:17.224289471 +0200
@@ -48,12 +48,14 @@
     const line = '-'.repeat(80);
     const goJson = readFileSync('assets/go-licenses.json', 'utf8');
     const goModules = JSON.parse(goJson).map(({name, licenseText}) => {
-      return {name, body: formatLicenseText(licenseText)};
+      // return {name, body: formatLicenseText(licenseText)};
+      return {name, body: (licenseText || '')};
     });
     const jsModules = Object.keys(dependencies).map((packageName) => {
       const {licenses, licenseFile} = dependencies[packageName];
       const licenseText = (licenseFile && !licenseFile.toLowerCase().includes('readme')) ? readFileSync(licenseFile) : '[no license file]';
-      return {name: packageName, licenseName: licenses, body: formatLicenseText(licenseText)};
+      // return {name: packageName, licenseName: licenses, body: formatLicenseText(licenseText)};
+      return {name: packageName, licenseName: licenses, body: (licenseText || '')};
     });
     const modules = [...goModules, ...jsModules];
     const licenseTxt = modules.map(({name, licenseName, body}) => {
