From c125868b5a86ed8ee6acf3482189cb7025ef7a28 Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Mon, 30 Jun 2025 18:09:00 +0200
Subject: [PATCH] =?UTF-8?q?webpack:=20Don=E2=80=99t=20use=20esbuild-loader?=
 =?UTF-8?q?/esbuilder?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Nils Philippsen <nils@redhat.com>
---
 webpack.config.js | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/webpack.config.js b/webpack.config.js
index 4662a30db5..38a91cfb2f 100644
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -4,7 +4,6 @@ import {init as licenseChecker} from 'license-checker-rseidelsohn';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import MonacoWebpackPlugin from 'monaco-editor-webpack-plugin';
 import {VueLoaderPlugin} from 'vue-loader';
-import EsBuildLoader from 'esbuild-loader';
 import {parse, dirname} from 'node:path';
 import webpack from 'webpack';
 import {fileURLToPath} from 'node:url';
@@ -15,7 +14,6 @@ import tailwindConfig from './tailwind.config.js';
 import tailwindcssNesting from 'tailwindcss/nesting/index.js';
 import postcssNesting from 'postcss-nesting';
 
-const {EsbuildPlugin} = EsBuildLoader;
 const {SourceMapDevToolPlugin, DefinePlugin} = webpack;
 const formatLicenseText = (licenseText) => wrapAnsi(licenseText || '', 80).trim();
 
@@ -149,15 +147,6 @@ export default {
     },
   },
   optimization: {
-    minimize: isProduction,
-    minimizer: [
-      new EsbuildPlugin({
-        target: 'es2020',
-        minify: true,
-        css: true,
-        legalComments: 'none',
-      }),
-    ],
     splitChunks: {
       chunks: 'async',
       name: (_, chunks) => chunks.map((item) => item.name).join('-'),
@@ -177,19 +166,6 @@ export default {
           },
         },
       },
-      {
-        test: /\.(js|ts)$/i,
-        exclude: /node_modules/,
-        use: [
-          {
-            loader: 'esbuild-loader',
-            options: {
-              loader: 'ts',
-              target: 'es2020',
-            },
-          },
-        ],
-      },
       {
         test: /\.css$/i,
         use: [
-- 
2.50.0

